apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/AbstractR4ResourceProvider.java

```diff
@@ -277,25 +278,39 @@ public T read(@IdParam IdType claimId, RequestDetails requestDetails) {
   @Trace
   public Bundle findByPatient(
       @RequiredParam(name = "mbi")
-          @Description(shortDefinition = "The patient identifier to search for")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.PAC_MBI,
+              value = OpenAPIContentProvider.PAC_MBI_VALUE)
           ReferenceParam mbi,
       @OptionalParam(name = "type")
-          @Description(shortDefinition = "A list of claim types to include")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.EOB_CLAIM_TYPE_SHORT,
+              value = OpenAPIContentProvider.EOB_CLAIM_TYPE_VALUE)
           TokenAndListParam types,
```

Comment:
This `type` attribute is actually in reference to `fiss` and `mcs` claim types for partially adjudicated data.